sample.yml

```diff
@@ -4,6 +4,19 @@
 # on components in your cluster!
 - include: "{{ playbook_dir }}/playbooks/check-requirements.yml"
 
+# ENABLE REPOS - all of our packages are installed from these repos, we need to
+# enable them before any roles can be used.
+- hosts: all
+  tasks:
+    - name: enable yum repos
+      sudo: yes
+      copy:
+        src: "roles/common/files/{{ item }}"
+        dest: "/etc/yum.repos.d/{{ item }}"
+      with_items:
+        - asteris-mantl-rpm.repo
+        - ciscocloud-rpm.repo
+
 # BASICS - we need every node in the cluster to have common software running to
```

Comment:
I'm glad to get feedback on this, since I wasn't really sure where to put it. I don't have any kind of a preference. I think that we _should_ be able to expect that folks will update their `mantl.yml` with new stuff in `sample.yml` every release (thinking about past releases when we have added/removed roles, etc.), but I'm also just generally unsure of where this belongs. 

I like the idea of using Ansible role dependencies more throughout our project so we don't have to depend on, say, the ordering of tasks in `sample.yml`. If we create a new role, maybe we can add it as a dependency for any role that uses our repos.